app/components/table/TableBodyPrivateProducts.tsx

```diff
@@ -89,9 +91,7 @@ export default function TableBodyPrivateProducts({ rows, isLoading = false }: Ta
               {row.activeRequest && <ActiveRequestBox data={{ ...row.activeRequest, cloud: 'private-cloud' }} />}
             </div>
 
-            <div className="md:col-span-2 lg:col-span-2">
-              {row.isTest && <TestProductBox data={{ createdAt: row.createdAt }} />}
-            </div>
+            <div className="lg:col-span-1 hidden lg:block"></div>
```

Comment:
can we revisit the `grid & columns` to have the structure as before?

Reply:
actually it is the structure "as before", copied from history